gdk/common/recipe_file/Recipe.py

```diff
@@ -0,0 +1,49 @@
+import json
+import yaml
+from pathlib import Path
+
+
+class Recipe:
+    def write(self, file_path: Path, content: dict) -> None:
```

Comment:
This is not very Open Closed this is open to modifications and closed to extensions. I am ok with this though given it is only 2 but we might want to refactor this later on